musicbrainzngs/musicbrainz.py

```diff
@@ -1259,8 +1259,10 @@ def submit_tags(**kwargs):
     """Submit user tags.
     Takes parameters named e.g. 'artist_tags', 'recording_tags', etc.,
     and of the form:
+    {entity_id1: {tag1: vote, ...}, ...}
```

Comment:
The documentation should be updated to say what the value of "vote" can be - "upvote", "downvote", or "withdraw". I wonder if it also makes sense to support alternate values for for this? Integers? 1, 0, -1? Booleans? True, False, None?

Reply:
Thanks for your comments. I will try to update this branch soon.

And I need to check, if the code still works after all those years.

Reply:
I improved the documentation now. Is it better now?

For the time being, I would stick with the official values for a vote ("upvote", "downvote", "withdraw"). At the moment, I'm not convinced to include a logic to convert boolean of interger values to this specific strings.